tests/integration_test.py

```diff
@@ -179,8 +185,52 @@ def test_src_fingerprint_no_object_specified(title, cmd_args, expected_output_re
         )
     ]
 )
-def test_src_fingerprint_on_org(title, cmd_args, expected_output_repos):
-    run_src_fingerprint(provider="github", args=["--object", "gg-src-fingerprint-org"]+cmd_args)
+def test_src_fingerprint_github_on_org(title, cmd_args, expected_output_repos):
+    run_src_fingerprint(provider="github", token=GH_INTEGRATION_TESTS_TOKEN, args=["--object", "gg-src-fingerprint-org"]+cmd_args)
     output_repos = get_output_repos("fingerprints.jsonl")
     os.remove("fingerprints.jsonl")
     assert output_repos == expected_output_repos
+
+
+@pytest.mark.parametrize(
+    "title, cmd_args, expected_output_repos", [
+        (
+            "Get all repos accesible to integration tests token",
+            ["--limit", "10"], 
+            {
+                "main"
+            }
+        ),
+    ]
+)
+def test_src_fingerprint_bitbucket_no_object_specified(title, cmd_args, expected_output_repos):
+    output = run_src_fingerprint(
+        provider="bitbucket",
+        token = BITBUCKET_INTEGRATION_TESTS_TOKEN,
+        args=cmd_args+["--provider-url", BITBUCKET_INTEGRATION_TESTS_URL]
+    )
+    output_repos = get_output_repos("fingerprints.jsonl")
+    os.remove("fingerprints.jsonl")
+    assert len(output_repos) == 10
```

Comment:
`expected_output_repos` could be replaced with `number_of_expected_output_repos` to make the test parametrizable.